src/wasm-config.h.in

```diff
@@ -131,18 +131,25 @@ __inline unsigned long wasm_clz_u64(unsigned __int64 mask) {
 }
 
 __inline unsigned long wasm_ctz_u32(unsigned long mask) {
-  unsigned long index;
-  _BitScanForward(&index, mask);
-  return sizeof(unsigned long) * 8 - (index + 1);
+    unsigned long index;
+    _BitScanForward(&index, mask);
+    return index;
```

Comment:
whoops, I looked at the CL where I added this -- looks like I just copy/pasted from wasm_clz_u32 :)